sandbox-apps/ecommerce-webapp/data/docker-compose.yml

```diff
@@ -105,8 +106,6 @@ services:
     environment:
       - FLASK_APP=discounts.py
       - FLASK_DEBUG=1
-      - POSTGRES_PASSWORD
```

Comment:
curious, why are we removing this vs. updating it to be ...

```
      - POSTGRES_PASSWORD=__PG_USER__
      - POSTGRES_USER=__PG_PASS__
```

i'll approve to unblock merging, but figured i'd flag this just to make sure.

Reply:
hm, you have a point there. That must have been a mistake of mine. Let me re-add those !